cerberus/tests/test_normalization.py

```diff
@@ -170,6 +170,15 @@ def test_nullables_dont_fail_coerce():
     assert_normalized(document, document, schema)
 
 
+def test_nullables_fail_coerce_on_non_null_values(validator):
+    schema = {'foo': {'coerce': lambda x: 1 / 0, 'nullable': True, 'type': 'integer'}}
+    document = {'foo': None}
```

Comment:
this is already covered in the previous test.

Reply:
True, but I feel the repetition is a good idea here because the new behavior is about the difference between these two cases (the value is `None` or not).

Reply:
i'm not saying you're wrong, but we don't do that elsewhere (oh dear, formulating this makes me feel old). i'm gonna amend this when merging.